proto/chat/v1beta/chat.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
+package spn.chat.v1beta1;
+
+option go_package = "github.com/tendermint/spn/x/chat/types";
+
+import "google/protobuf/any.proto";
+import "gogoproto/gogo.proto";
+
+message Channel {
+  int32 id = 1;
+  bytes creator = 2 [(gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.AccAddress"];
+  string name = 3; 
+  string subject = 4; 
+  int32 messageCount = 5; 
+  google.protobuf.Any metadata = 6;
+}
+
+message Message {
+  int32 channelId = 1;
+  int32 messageNb = 2;
+  bytes author = 3 [(gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.AccAddress"];
+  string content = 4; 
+  repeated string tags = 5;
+  int64 timestamp = 6;
+  message UserVote {
+    bytes user = 1 [(gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.AccAddress"];
+    Vote vote = 2;
+  }
+  repeated UserVote userVotes = 7;
```

Comment:
maybe we should consider using [oneOf](https://developers.google.com/protocol-buffers/docs/proto3#oneof) here. For now, a _Message_ type can have either a `string Text` content or a `Pool poll` content. But in future this can be extended to `call`, `location`, etc.

also related to https://github.com/tendermint/spn/pull/2/files#diff-4b2f4dd5bc8925c44a7b8337dc4a282038b106f9aa277fe5041a60a196590d69R22.